### ISSUE TITLE
Add ability to decrypt every file in a save file.

### ISSUE THREAD

**Issue by TailsGuy** [bucanero/apollo-ps3#130]
Super Street Fighter IV has about 180 files in its save, which is [if you didn't know] humongous. It'd be great if I could decrypt all of these saves and not resort to Bruteforce and see if that works.

**Comment by alebustelo** [comment]
I second this, a simple "select all" option would make this a lot less tedious. In fact, a batch decrypt of all save files in, say, dev_usb000 would be appreciated as well.

Yes, it would be nice if the emulator supported the original encryption, but in the meantime, this doesn't sound like a crazy request.

**Comment by TailsGuy** [comment]
Yes, it's just convenient. Even if you take RPCS3 out of the equation, for folks who want to edit or study these saves and having them decrypted is a necessity, it's a really long hassle to have to manually go through 90 or so save files.

**Comment by Wild-Dog** [comment]
I would really appreciate it personally. A bulk option to decrypt and import decrypted saves would be fantastic.

I just wanted to try playing my old PS3 games on my gaming PC.
